docs/conf.py

```diff
@@ -70,12 +70,31 @@
     ("py:class", "sqlalchemy.orm.decl_api.DeclarativeMeta"),
     ("py:class", "sqlalchemy.sql.sqltypes.TupleType"),
     ("py:class", "sqlalchemy.dialects.postgresql.named_types.ENUM"),
+    ("py:class", "_orm.Mapper"),
+    ("py:class", "_orm.registry"),
+    ("py:class", "_schema.MetaData"),
+    ("py:class", "_schema.Table"),
+    ("py:class", "_RegistryType"),
+    ("py:class", "abc.Collection"),
+    ("py:class", "TypeEngine"),
+    ("py:class", "ExternalType"),
+    ("py:class", "UserDefinedType"),
```

Comment:
These should definitely work. What was the issue with them?

Reply:
I got the following error when these weren't included:
```
/home/oracle/Code/Starlite/litestar/litestar/contrib/repository/filters.py:docstring of litestar.contrib.repository.filters.CollectionFilter.values:1: WARNING: py:class reference target not found: abc.Collection
/home/oracle/Code/Starlite/litestar/litestar/contrib/repository/filters.py:docstring of litestar.contrib.repository.filters.CollectionFilter.__init__:1: WARNING: py:class reference target not found: abc.Collection
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/base.py:docstring of litestar.contrib.sqlalchemy.base:1: WARNING: py:class reference target not found: _schema.MetaData
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/base.py:docstring of litestar.contrib.sqlalchemy.base:1: WARNING: py:class reference target not found: _schema.Table
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/base.py:docstring of litestar.contrib.sqlalchemy.base.AuditBase.registry:1: WARNING: py:class reference target not found: _RegistryType
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/base.py:docstring of litestar.contrib.sqlalchemy.base:1: WARNING: py:class reference target not found: _orm.registry
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/base.py:docstring of litestar.contrib.sqlalchemy.base:1: WARNING: py:class reference target not found: _orm.Mapper
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/base.py:docstring of litestar.contrib.sqlalchemy.base:1: WARNING: py:class reference target not found: _schema.MetaData
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/base.py:docstring of litestar.contrib.sqlalchemy.base:1: WARNING: py:class reference target not found: _schema.Table
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/base.py:docstring of litestar.contrib.sqlalchemy.base.Base.registry:1: WARNING: py:class reference target not found: _RegistryType
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/base.py:docstring of litestar.contrib.sqlalchemy.base:1: WARNING: py:class reference target not found: _orm.registry
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/base.py:docstring of litestar.contrib.sqlalchemy.base:1: WARNING: py:class reference target not found: _orm.Mapper
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/types.py:docstring of litestar.contrib.sqlalchemy.types:1: WARNING: py:class reference target not found: ExternalType
/home/oracle/Code/Starlite/litestar/docstring of litestar.contrib.sqlalchemy.types.GUID.cache_ok:100: WARNING: py:class reference target not found: ExternalType
/home/oracle/Code/Starlite/litestar/docstring of litestar.contrib.sqlalchemy.types.GUID.cache_ok:100: WARNING: py:class reference target not found: UserDefinedType
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/types.py:docstring of litestar.contrib.sqlalchemy.types:1: WARNING: py:class reference target not found: TypeEngine
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/types.py:docstring of litestar.contrib.sqlalchemy.types:1: WARNING: py:meth reference target not found: type_engine
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/types.py:docstring of litestar.contrib.sqlalchemy.types:1: WARNING: py:class reference target not found: _types.TypeDecorator
/home/oracle/Code/Starlite/litestar/.venv/lib/python3.11/site-packages/sqlalchemy/sql/sqltypes.py:docstring of sqlalchemy.sql.sqltypes.CHAR:1: WARNING: py:meth reference target not found: _types.TypeDecorator.process_result_value
/home/oracle/Code/Starlite/litestar/litestar/contrib/sqlalchemy/types.py:docstring of litestar.contrib.sqlalchemy.types:1: WARNING: py:class reference target not found: _types.TypeDecorator
/home/oracle/Code/Starlite/litestar/.venv/lib/python3.11/site-packages/sqlalchemy/sql/sqltypes.py:docstring of sqlalchemy.sql.sqltypes.CHAR:1: WARNING: py:meth reference target not found: _types.TypeDecorator.process_bind_param
```

Reply:
Hm. I need to look into that. Those should resolve, either natively or with our custom extension or be silenced by our custom extension. Let's just keep them on the suppression list for now.